src/pkg/selectors/sharepoint.go

```diff
@@ -407,6 +407,30 @@ func (s SharePointScope) setDefaults() {
 	}
 }
 
+// DiscreteCopy makes a shallow clone of the scope, then replaces the clone's
+// site comparison with only the provided site.
+func (s SharePointScope) DiscreteCopy(site string) SharePointScope {
+	return discreteCopy(s, site)
+}
+
+// ---------------------------------------------------------------------------
+// Backup Details Filtering
+// ---------------------------------------------------------------------------
+
+// Reduce filters the entries in a details struct to only those that match the
```

Comment:
Weird diff: this func wasn't changed.  What changed is that `matchesInfo()` was moved below it.